lib/fetchElement.js

```diff
@@ -0,0 +1,6 @@
+export async function fetchElement() {
```

Comment:
Maybe rename the function to `fetchCharacters` to make it more descriptive. 
```suggestion
export async function fetchCharacters() {
```